compatibility/history-proxy/.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
What exactly that?
I think It wont run, because the github actions just work in .github/workflows. (unless something changed)